primitiv/CMakeLists.txt

```diff
@@ -103,6 +103,16 @@ endif()
 
 # Build rules of the OpenCL backend.
 if(PRIMITIV_USE_OPENCL)
+  add_custom_command(OUTPUT opencl_device_kernel.h
```

Comment:
`opencl_device_kernel.h` -> `${CMAKE_CURRENT_BINARY_DIR}/opencl_device_kernels.inc`.
The extension `.h` should represent *header* files which describe APIs of the library.